pkg/netdevice/netDeviceProvider.go

```diff
@@ -86,7 +86,7 @@ func (np *netDeviceProvider) AddTargetDevices(devices []*ghw.PCIDevice, deviceCo
 			// exclude netdevice in-use in host
 			if isDefaultRoute, _ := hasDefaultRoute(device.Address); !isDefaultRoute {
 				aPF := utils.IsSriovPF(device.Address)
-				if aPF && utils.SriovConfigured(device.Address) {
+				if aPF {
```

Comment:
We currently use sriov-device-plugin to assign PF to container for Baremetal and VM use-cases
with this change, it will cause device plugin to skip PFs. some Mellanox nics come preconfigured with SR-IOV enabled and 8 VFs although not used.

Reply:
@adrianchiris will you prefer the second solution?

https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/408

Reply:
> We currently use sriov-device-plugin to assign PF to container for Baremetal and VM use-cases with this change, it will cause device plugin to skip PFs. some Mellanox nics come preconfigured with SR-IOV enabled and 8 VFs although not used.

@adrianchiris I recall that we didn't claim PF support. I think this feature should be documented if we all agree to support PF use case in sriov-network-device-plugin, otherwise it may be ignored by others who contribuing to this project. wdyt?

Reply:
I vote for @SchSeba #408 solution since with the default setting it does not have any change

Reply:
> @adrianchiris I recall that we didn't claim PF support. I think this feature should be documented if we all agree to support PF use case in sriov-network-device-plugin, otherwise it may be ignored by others who contribuing to this project. wdyt?

I recall we had discussion on it a while ago, where we discussed the use-case which is similar to VM use-case where you use DP to assign a PCI device without it having a parent.

agree, I will some documentation and we can discuss there.